cffdump/scripts/texturator-to-unit-test-5xx.lua

```diff
@@ -0,0 +1,200 @@
+-- Parse logs from https://github.com/freedreno/freedreno/
+-- test-texturator.c to generate a src/freedreno/fdl/fd6_layout_test.c
```

Comment:
nit: I guess that would be fd5_layout_test.c (not that it exists yet.. although a5xx could do vulkan if someone showed up who had hw and cared enough, so moving a5xx layout code it fdl is not unreasonable)

Reply:
lgtm, r-b